modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx

```diff
@@ -106,13 +106,13 @@ describe('View Channel', () => {
 			)
 		);
 
-		const {container} = render(<DefaultComponent />);
+		const {container, debug} = render(<DefaultComponent />);
```

Comment:
```suggestion
		const {container} = render(<DefaultComponent />);
```

hey @interaminense please remove this "debug", ok? Thanks!

Reply:
Done. Thanks!